packages/react-events/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const ReactEvents = require('./src/ReactEvents');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest.
+module.exports = ReactEvents.default || ReactEvents;
```

Comment:
I don't think we've been doing this on our new packages. See: https://github.com/facebook/react/blob/master/packages/react-cache/index.js

Reply:
Ah good catch

Reply:
Indeed! https://github.com/facebook/react/pull/15152